includes/class-sensei-settings.php

```diff
@@ -302,17 +302,14 @@ public function init_fields() {
 		);
 
 		// Course Settings.
-
-		if ( Sensei()->feature_flags->is_enabled( 'course_theme' ) ) {
-			$fields['sensei_learning_mode_all'] = array(
-				'name'        => __( 'Learning mode', 'sensei-lms' ),
-				'description' => __( 'Enable this mode for your courses to show an immersive and dedicated view for the course, lessons, and quiz.*', 'sensei-lms' ),
-				'form'        => 'render_learning_mode_setting',
-				'type'        => 'checkbox',
-				'default'     => false,
-				'section'     => 'course-settings',
-			);
-		}
+		$fields['sensei_learning_mode_all'] = array(
+			'name'        => __( 'Learning mode', 'sensei-lms' ),
+			'description' => __( 'Enable this mode for your courses to show an immersive and dedicated view for the course, lessons, and quiz.*', 'sensei-lms' ),
```

Comment:
I must have not pay attention to the copy when I added this. But it should be `quizzes` at the end of description.

Reply:
👍 Updated here: https://github.com/Automattic/sensei/pull/4662/commits/73b9b1fe0fe1810899950a8e87e12fe9fcfec6dc

I also changed updated the description to use that variable, so we don't duplicate the text. I also removed the `*`. I think that's not necessary, right?

Reply:
Yep. Don't think we need it.